scripts/paas_app_wrapper.sh

```diff
@@ -26,7 +26,11 @@ case $NOTIFY_APP_NAME in
     ;;
   delivery-worker-periodic)
     scripts/run_app_paas.sh celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=2 \
-    -Q periodic-tasks,statistics-tasks 2> /dev/null
+    -Q periodic-tasks 2> /dev/null
+    ;;
+  delivery-worker-reporting)
+    scripts/run_app_paas.sh celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=2 \
```

Comment:
Can we have more workers on this app?

Reply:
i'll put it up to 11 like all the other apps